composer.json

```diff
@@ -17,12 +17,12 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0",
+        "php": ">=7.3 || >=8.0 ",
```

Comment:
I'd normally prefer something like `^7.3 || ^8.0` here (to prevent installation on PHP 9 if the package hasn't been tested on it), but I'm okay keeping this as-is for this package.